GSA/made-in-america
# [MIA-473] Department of Labor url update

## Description

This PR updated the Dept of Labor to have the URL to their waiver.  
link to ticket -> https://cm-jira.usa.gov/browse/MIA-473

## How to test

- pull down branch 
- run `npm run start`
- go to browser 
- vist `http://localhost:4000/financial-assistance/`
- go to Department of Labor and make sure that the link opens to a new tab and goes to the right URL  

To prepare for code review, please follow the following checks.
